features/support/attachment_helper.rb

```diff
@@ -27,7 +27,10 @@ def create_external_attachment(url, attachment_title)
     click_on "Add new external attachment"
     fill_in "Title", with: attachment_title
     fill_in "External url", with: url
-    click_on "Save"
+
+    design_system = @user.has_permission? "Preview design system"
+    click_on design_system ? "Next" : "Save"
```

Comment:
I think the convention for test files is as follows:

def create_external_attachment(url, attachment_title)
  design_system = @user.has_permission? "Preview design system"

  click_on "Add new external attachment"
  fill_in "Title", with: attachment_title
  fill_in "External url", with: URL
  click_on design_system ? "Next" : "Save"
end

Reply:
OK, makes sense 👍